src-ui/p.html

```diff
@@ -446,6 +446,7 @@ <h2 id="title2">読み込み中です...</h2>
   <button type="button" class="btn" id="btnflush"   data-button-exec="flushexcell">__flushexcell__</button>
   <button type="button" class="btn" id="btnpresets" data-button-exec="applypreset">__applypreset__</button>
   <button type="button" class="btn" data-disp-pid="stostone" data-press-exec="dropblocks,resetblocks">__dropblocks__</button>
+  <button type="button" class="btn" data-disp-pid="slither" data-press-exec="outlineshaded">__outlineshaded__</button>
```

Comment:
Isn't it better to use `data-button-exec` instead? I'm not sure there's a benefit to applying it on mousedown instead of mouseup.

Reply:
oops, i didn't notice i had done that.  that explains why i ran into that bug with the event handling :)